File: clojure/clockhands.clj
(doseq [i (range 0 11)]
  (let [
    p (int (/ (* (+ i 0.5) 43200.0) 11))
    h (quot p 3600)
    m (rem p 3600)]
    (printf "%02d:%02d:%02d\n" (if (zero? h) 12 h) (quot m 60) (rem m 60))))

File: python/clockhands.py
for i in range(0, 11):
    t = (43200 * i + 21600) // 11
    h, t = divmod(t, 3600)
    m, s = divmod(t, 60)
    print(f'{(h or 12):02}:{m:02}:{s:02}')